### ISSUE TITLE
 "list index out of range" error when stopping someone else's exam mode

### ISSUE THREAD

**Issue by lemertens** [linuxmuster/linuxmuster-webui7#279]
If a teacher tries to deactivate the exam mode for one or more students that was activated by another teacher the process throws an error and stalls. The exam mode remains active

```

Serverfehler
Server error occured. This is likely a bug.
Request
PATCH /api/lmn/session/exam/stop
Type
Exception
Message
Error: list index out of range
Traceback

Traceback (most recent call last):
  File "/usr/lib/linuxmuster-webui/plugins/lmn_session_new/views.py", line 174, in handle_api_stop_exam
    lmn_getSophomorixValue(sophomorixCommand, 'COMMENT_EN')
  File "/usr/lib/linuxmuster-webui/plugins/lmn_common/api.py", line 133, in lmn_getSophomorixValue
    output = output.split('# JSON-begin')[1]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/aj/api/endpoint.py", line 77, in wrapper
    result = fx(self, context, *args, **kwargs)
  File "/usr/lib/linuxmuster-webui/plugins/lmn_session_new/views.py", line 176, in handle_api_stop_exam
    raise Exception('Error:\n' + str(e))
Exception: Error:
list index out of range


```

**Comment by kiarn** [comment]
Hello @lemertens,

Thanks, it should be solved in 7.2.71.

**Comment by lemertens** [comment]
Salut Arnaud,
bien fait, ca marche maintenant !
Merci !
